scripts/CEE/restart-dns-default/script.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+
+set -e
+set -o errexit
+set -o nounset
+set -o pipefail
+
+CURRENTDATE=$(date +"%Y-%m-%d %T")
+
+## validate input
+if [[ -z "${NAMESPACE}" ]]; then
+    echo "Variable NAMESPACE cannot be blank"
+    exit 1
+fi
+
+start_job(){
+    echo "Job started at $CURRENTDATE"
+    echo ".................................."
+    echo
+}
+
+finish_job(){
+    echo
+    echo ".................................."
+    echo "Job finished at $CURRENTDATE"
+}
+
+
+
+## Function for daemonsent rollout start
+restart_dns_pods(){
+
+    echo -e "\nRestarting dns_pods in \"${NAMESPACE}\" namespace. Wait until rollout reaches 5 out 5"
+    
```

Comment:
Looks you hard coded the `openshift-dns` but also use the variable `$NAMESPACE` which introduced the discrepancies. You can define the variable at the top of the script and assign the value `openshift-dns` to make the use consistent.

Reply:
hard-coded and removed from metadata

Reply:
@xiaoyu74 
regarding the comment 

`I can see you submitted the PR from your local main branch which is not the best practice, better to avoid submitting from main but creating a feature or topic branch (e.g restart_dns_pods) for the changes and submit a PR from there.`

Do you want me to close this PR and create a new one with the new branch, or leave as it is?

Reply:
@diakovnec - Sorry for my late response. Yeah, please create a new one with all the updates in the script and submit from your local feature/topic branch. Thank you.